src/main/js/helpers/styles.js

```diff
@@ -43,6 +43,9 @@ export default {
     datelineGroup: "carbon-dateline-group",
     datelinePoint: "carbon-dateline-point",
     dateline: "carbon-dateline",
+    noDataLayout: "no-data-layout",
```

Comment:
Can you create a consistent class name value here?

Reply:
Updated the names.